internal/manifests/var.go

```diff
@@ -113,6 +120,43 @@ func serviceNameCompactorHTTP(stackName string) string {
 	return fmt.Sprintf("loki-compactor-http-%s", stackName)
 }
 
+func serviceNameQueryFrontendGRPC(stackName string) string {
+	return fmt.Sprintf("loki-query-frontend-grpc-%s", stackName)
+}
+
+func serviceNameQueryFrontendHTTP(stackName string) string {
+	return fmt.Sprintf("loki-query-frontend-http-%s", stackName)
+}
+
 func fqdn(serviceName, namespace string) string {
 	return fmt.Sprintf("%s.%s.svc.cluster.local", serviceName, namespace)
 }
+
+// jobLabelForComponent is a servicemonitor.spec.jobLabel including the loki.grafana.com/component:<component> label
+func jobLabelForComponent(component string) string {
+	return fmt.Sprintf("loki.grafana.com/component: %s", component)
+}
```

Comment:
The `ServiceMonitor.Spec.JobLabel` fields only needs the value of the label key, e.g. `loki.grafana.com/component`. Prometheus will pick the value by reading this key from the time series and use it as the scrape job name.